### ISSUE TITLE
Blocked mirror for repositories http://maven.geotoolkit.org

### ISSUE THREAD

**Issue by ghobona** [Geomatys/geotoolkit#20]
I get the following error when I try to build a project that references geotoolkit, using Maven 3.8.1.

```
Blocked mirror for repositories: [geotoolkit (http://maven.geotoolkit.org/, default, releases+snapshots)]

Since Maven 3.8.1 http repositories are blocked.

Possible solutions:
- Check that Maven pom files not contain http repository http://maven.geotoolkit.org/
- Add a mirror(s) for http://maven.geotoolkit.org/ that allows http url in the Maven settings.xml
- Downgrade Maven to version before 3.8.1 in settings
```

My pom.xml file contains this:

```xml
<repositories>
  <repository>
    <id>geotoolkit</id>
    <name>Geotk Modules</name>
    <url>http://maven.geotoolkit.org/</url>
    <layout>default</layout>
  </repository>
</repositories>

```

I tried replacing the url with:

* https://maven.geotoolkit.org
* https://nexus.geomatys.com/repository/geotoolkit

None of the URLs listed above worked either.

**Comment by desruisseaux** [comment]
Hello Gobe. Can you try replacing "http" by "https" in the `pom.xml` file? Also, can you check if a browser can open the https://maven.geotoolkit.org/ URL?

**Comment by desruisseaux** [comment]
Sorry I missed the part of the comment saying that you already tried with "https". I presume that after making this change, the error message saying _"Since Maven 3.8.1 http repositories are blocked"_ changed to something else. What is the new error message?

**Comment by ghobona** [comment]
Interestingly, when I reload the maven project. I get the original error message: "Blocked mirror for repositories: [geotoolkit (http://maven.geotoolkit.org, default, releases+snapshots)]". This happens even though the repository is https://maven.geotoolkit.org

<img width="1438" alt="Screenshot 2022-02-21 at 21 09 49" src="https://user-images.githubusercontent.com/18121038/155026766-3921b48f-3617-4b59-97ee-7a46399f05bf.png">

**Comment by desruisseaux** [comment]
The first screenshot contains a link to https://github.com/opengeospatial/ets-ogcapi-edr10. Is it the project that we try to build? If yes I can investigate directly on the project and submit pull request if this is okay for you. Or if the issue is with another project, is there a link that I could checkout?

**Comment by ghobona** [comment]
@desruisseaux Yes, the project is https://github.com/opengeospatial/ets-ogcapi-edr10

It should build with `mvn clean install site -Pintegration-tests -Dsource=8`

**Comment by desruisseaux** [comment]
I just tried to build the project from source. When using Java 17, the build fails with an error unrelated to Geotk. The error seems to come from `org.mockito.Mockito`:

```
java.lang.ClassFormatError accessible: module java.base does not "opens java.lang" to unnamed module @1a2dd1b8
	at org.opengis.cite.ogcapiedr10.VerifySuiteFixtureListener.setUpClass(VerifySuiteFixtureListener.java:32)
```

I suspect that upgrading the Mockito dependency would fix above issue. In the meantime I tried to build with Java 8 and it worked. I do not know why the maven.geotoolkit.org repository seems blocked on your side…

Looking at the source code of EDR project, I saw that Geotk is used for referencing services. In recent Geotk versions, referencing services are now handled by Apache SIS. Would you be interested in a patch replacing Geotk by Apache SIS? An advantage is that Apache SIS is deployed on Maven Central repository, so we could remove the maven.geotoolkit.org repository completely. This proposal assumes that the EDR project is not used in the same Java Virtual Machine than other OGC projects using Geotk, otherwise we would have conflicts unless we perform the same upgrade on all projects that are run together.

**Comment by desruisseaux** [comment]
I created a pull request replacing Geotk dependency by Apache SIS: https://github.com/opengeospatial/ets-ogcapi-edr10/pull/97

If this pull request is accepted, then EDR would not depend anymore on maven.geotoolkit.org repository.
